src/types.ts

```diff
@@ -479,6 +479,7 @@ export interface DomainMetadata {
 export enum IPFSGatewayUri {
   ipfs = "ipfs.io",
   fleek = "ipfs.fleek.co",
```

Comment:
is fleek still being used? if not we should remove this to avoid confusion

Reply:
It can still be used as a gateway.